UI/DamageIndicator.cs

```diff
@@ -0,0 +1,44 @@
+using Godot;
+using System;
```

Comment:
Unused import?

Also SkyOfSteel convention is two blank lines between imports and the file contents (as well as between functions ect)